### DISCUSSION TITLE
【Step-1】请各位队长开始新建团队

### DISCUSSION THREAD

**Discussion by GreyZeng** [FZUSESPR21/Class_Resources#3] [General]
各位队长请注意，请开始新建自己的团队，具体做法如下：

队长在这个页面：
https://github.com/orgs/FZUSESPR21/teams

![image](https://user-images.githubusercontent.com/12782103/112237810-613ba980-8c7e-11eb-8596-8bc58a1add63.png)

点击 New team，输入如下信息

![image](https://user-images.githubusercontent.com/12782103/112237922-a069fa80-8c7e-11eb-8662-b61a781a40db.png)

新建Team以后，进入自己新建的team，在你的team的这个位置，介绍一下你的团队，以下是示例：

![image](https://user-images.githubusercontent.com/12782103/112238764-3baf9f80-8c80-11eb-9c18-608a650c0667.png)

介绍的风格可以自定义，

但是必须包括 队员的**学号，Github地址，博客地址，岗位职责** 信息

完成上述操作后，请把你们团队的team链接留言到这个帖子下面。

截止时间：2021-03-24 晚上10点

逾期或者未按上述要求留言的，会扣除一部分团队分数

注：这次的团队是你们之前上周二组好的哈，不是要你们重新组队

**Comment by fzu-kwz** [comment]
https://github.com/orgs/FZUSESPR21/teams/team-4

**Comment by Lyr-GW** [comment]
https://github.com/orgs/FZUSESPR21/teams/team-5

**Comment by Tozzger** [comment]
https://github.com/orgs/FZUSESPR21/teams/team-7

**Comment by hb594709591** [comment]
[https://github.com/orgs/FZUSESPR21/teams/team-6](https://github.com/orgs/FZUSESPR21/teams/team-6)

**Comment by LinxuanZheng** [comment]
https://github.com/orgs/FZUSESPR21/teams/team-2

**Reply by GreyZeng** [reply]
> （逐梦校友圈）https://github.com/orgs/FZUSESPR21/teams/team-1

发错位置了。

**Comment by Fzuerzmj** [comment]
（逐梦校友圈）https://github.com/orgs/FZUSESPR21/teams/team-1

**Comment by gz321** [comment]
https://github.com/orgs/FZUSESPR21/teams/team-3

**Comment by Honghonglin** [comment]
https://github.com/orgs/FZUSESPR21/teams/tut

**Comment by Hao23333** [comment]
[https://github.com/orgs/FZUSESPR21/teams/team](https://github.com/orgs/FZUSESPR21/teams/team)
